ballet/util/fs.py

```diff
@@ -71,3 +77,88 @@ def isemptyfile(filepath):
         return filesize == 0
     else:
         return False
+
+
+def ispathlike(obj):
```

Comment:
What are the benefits of doing this over just calling isinstance?

Reply:
none, fixing

it used to have more functionality before i simplified code